app/design/src/main/content/jcr_root/etc/designs/example/publish/src/styles/main.scss

```diff
@@ -0,0 +1,5 @@
+@import 'module';
```

Comment:
Happy[js]: Version: 5.0.0. Threads: 4
Happy[js]: All set; signaling webpack to proceed.
[hardsource:0a693d4f] Writing new cache 0a693d4f...
[hardsource:0a693d4f] Tracking node dependencies with: package.json, node_modules/*.

styles/_module.scss
1:1     ÔÇ╝  Unexpected empty source (no-empty-source) [stylelint]


styles/main.scss
2:23    ÔÇ╝  Expected newline before "}" of a multi-line block (block-closing-brace-newline-before) [stylelint]

Hash: ff054a7621faa9802378
Version: webpack 4.17.2

Reply:
![image](https://user-images.githubusercontent.com/10673561/45938836-fc7c3700-bfcd-11e8-826b-17a6472b47cc.png)

main.css is at bad level. clientlib is referencing file dist/main.css not dist/css/main.css